src/mixins/filing-template-mixin.ts

```diff
@@ -306,23 +304,46 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
   }
 
   /**
-   * Prepares name translations for non-draft save.
-   * @returns the updated name translations array
+   * Builds a change of registration filing from store data. Used when saving a filing.
+   * @param isDraft whether this is a draft
+   * @returns the change filing body
    */
-  private prepareNameTranslations () : NameTranslationIF[] {
-    // Filter out and modify name translation to match schema
-    return this.getNameTranslations?.filter(x => x.action !== ActionTypes.REMOVED)
-      .map(x => {
-        let nameTranslation = {
-          name: x.name
-        }
-        if (x.action !== ActionTypes.ADDED) {
-          nameTranslation['id'] = x.id
+  buildChangeFiling (isDraft: boolean): ChangeFirmIF {
```

Comment:
You can get rid of these Codecov comments by writing a unit test that executes this code. Example:
https://github.com/bcgov/business-create-ui/blob/9c73415ce3472859096ed199e28dab84d14a33b6/tests/unit/filing-template-mixin.spec.ts#L32

Reply:
This was really cool, implemented the start of this in Edit UI, thank you!

Reply:
Looks good 👍